src/parse.zig

```diff
@@ -171,10 +172,13 @@ pub const Node = struct {
         base: Node = Node{ .tag = Tag.value, .tree = undefined },
         start: ?TokenIndex = null,
         end: ?TokenIndex = null,
+        string_value: ArrayList(u8) = undefined,
```

Comment:
Given that we don't store an allocator as part of the `Node` or `Node.Value` structs, it would be better to use `std.ArrayListUnmanaged(u8)` here which would not force you to set it to poison with `undefined`:

```zig
string_value: std.ArrayListUnmanaged(u8) = .{},
```